sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -71,89 +70,178 @@ public class Devices
+    public virtual Response<BulkRegistryOperationResult> CreateIdentitiesWithTwin(IDictionary<DeviceIdentity, TwinData> devices, CancellationToken cancellationToken = default)
 
     /// <summary>
-    /// Create multiple device identities. A maximum of 100 creations can be done per call, and each device identity must be unique. For larger scale operations, consider using IoT Hub jobs (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).
+    /// Create multiple devices. A maximum of 100 creations can be done per call, and each device identity must be unique. For larger scale operations, consider using IoT Hub jobs (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).
     /// </summary>
     /// <param name="deviceIdentities">The devices to create.</param>
     /// <param name="cancellationToken">The cancellation token.</param>
     /// <returns>The result of the bulk operation.</returns>
     public virtual async Task<Response<BulkRegistryOperationResult>> CreateIdentitiesAsync(IEnumerable<DeviceIdentity> deviceIdentities, CancellationToken cancellationToken = default)
 
     /// <summary>
-    /// Update multiple device identities. A maximum of 100 updates can be done per call, and each operation must be done on a different identity. For larger scale operations, consider using IoT Hub jobs (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).
+    /// Create multiple devices. A maximum of 100 creations can be done per call, and each device identity must be unique. For larger scale operations, consider using IoT Hub jobs (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).
+    /// </summary>
+    /// <param name="deviceIdentities">The devices to create.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>The result of the bulk operation.</returns>
+    public virtual Response<BulkRegistryOperationResult> CreateIdentities(IEnumerable<DeviceIdentity> deviceIdentities, CancellationToken cancellationToken = default)
+
+    /// <summary>
+    /// Update multiple devices. A maximum of 100 updates can be done per call, and each operation must be done on a different identity. For larger scale operations, consider using IoT Hub jobs (https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities).
     /// </summary>
     /// <param name="deviceIdentities">The devices to update.</param>
-    /// <param name="force">If true, the devices will be updated even if their ETag is out of date.
-    /// If false, each device will only be updated if its ETag is up to date.</param>
+    /// <param name="precondition">Perform these operations only on this condition.</param>
```

Comment:
nit: The text for `precondition` should be similar to what we have in other methods